internal/subshell/cmd/cmd.go

```diff
@@ -96,7 +97,12 @@ func (v *SubShell) CleanUserEnv(cfg sscommon.Configurable, envType sscommon.RcId
 	// Clean up old entries
 	oldEnv := cfg.GetStringMap(envType.Key)
 	for k, v := range oldEnv {
-		if err := cmdEnv.unset(k, v.(string)); err != nil {
+		val, ok := v.(string)
+		if !ok {
+			logging.Debug("Invalid non-string value in environment mapping")
+			continue
+		}
```

Comment:
You ran into this? What was the value? I would think there's no version of this where the values aren't strings.

Reply:
I didn't run into this but it is a potential panic that we can handle here. I think this is better than relying on the `map[string]interface{}` being what we expect.